requirements.in

```diff
@@ -1,5 +1,6 @@
 coverage==4.5.1
-Django==1.11.23
+ddt
+Django<2
```

Comment:
does this Django version correspond to this PR

Reply:
What do you mean?
I think that we should use Django<2 instead of django==1.11.23. 
Django 1.11.23 is old, the latest 1.11.* version is the 1.11.29 version

Reply:
My fault, I read Django>2